src/core/sys/dragonflybsd/netinet/in_.d

```diff
@@ -0,0 +1,551 @@
+/**
+ * D header file for DragonFlyBSD's extensions to POSIX's netinet/in.h.
+ *
+ * Copyright: Copyright 2017 -
+ * License:   $(WEB www.boost.org/LICENSE_1_0.txt, Boost License 1.0).
+ * Authors: Jonathan M Davis, Diederik de Groot(port:DragonFlyBSD)
+ * Copied:  From core/sys/freebsd/sys
+ */
+module core.sys.dragonflybsd.netinet.in_;
+
+import core.sys.dragonflybsd.sys.cdefs;
+
+public import core.sys.posix.netinet.in_;
+
+version(DragonFlyBSD):
```

Comment:
Why don't you move this five lines higher?

Reply:
Done thanks !